eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -584,6 +584,13 @@ jobs:
               extraBuildProperties="$extraBuildProperties /p:DotNetSourceOnlyPostBuildSign=true"
             fi
 
+            if [[ '${{ length(parameters.reuseBuildArtifactsFrom) }}' -gt '0' ]]; then
+               # Extract the prebuilt packages to a temporary directory
+               mkdir -p "$(Agent.TempDirectory)/prebuilt-packages"
+               tar -xzf $(sourcesPath)/prereqs/packages/archive/Private.SourceBuilt.Artifacts.*.tar.gz -C "$(Agent.TempDirectory)/prebuilt-packages"
+               customBuildArgs="$customBuildArgs --with-packages $(Agent.TempDirectory)/prebuilt-packages/"
```

Comment:
While these are a type of prebuilt-package, I think it would be better to call these previously-built-packages, psb, or previous-packages.  The system also has the ability to input prebuilt packages which are very different that previously built packages.